### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by FlaviaCohen**
## Links:
https://trello.com/c/cWK9ZDuc/15-3-as-a-user-i-should-be-able-to-login-so-i-can-use-my-target-mvd-account

## What & Why:
As a user I should be able to sign in so my account is created in Target MVD
The signin feature was already implemented but /sign-in page had no style and there were some texts missing.
I styled the page in order to give a better UX in desktop and mobile version.

width: 1920
Screen Shot 2022-03-30 at 17 30 11

Error case: 
Screen Shot 2022-03-30 at 17 35 23

Mobile width: 
Screen Shot 2022-03-30 at 17 35 42

Mobile width: 
Screen Shot 2022-03-30 at 17 35 51

### CODE REVIEW

File: src/pages/Login/index.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ import useTranslation from 'hooks/useTranslation';
 import routesPaths from 'routes/routesPaths';
 import Input from 'components/form/Input';
 import Button from 'components/common/Button';
+import smiles from '../../assets/smiles.svg';
```
Comment (Stukz) [inline src/pages/Login/index.js]: I'm curious if you can do 
```suggestion
import smiles from 'assets/smiles.svg';
```

**Review by Stukz — APPROVED** [review]
LGTM

File: src/pages/Login/index.js (hunk 1 above)
Comment (FlaviaCohen) [reply src/pages/Login/index.js]: Oh yes!! It worked. Thank you.

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-14.17.0
+14.17.0
```
Comment (alejobh) [inline .nvmrc]: revert this EOF

File: src/index.jsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { HelmetProvider } from 'react-helmet-async';
 
 import { SUPPORTED_LANGUAGES, DEFAULT_LANGUAGE } from './constants/constants';
 import store from 'state/store';
-import App from './App';
+import App from './App.jsx';
```
Comment (alejobh) [inline src/index.jsx]: mmm if you have to add .jsx extension in the import, maybe there is something wrong with the project config

File: src/pages/Login/index.jsx [hunk 4]
```diff
@@ -51,36 +52,50 @@ const Login = () => {
...
+          start conecting with others who share your interest.
+        </p>
+      </div>
+      <div className="login__form-container">
+        <form className="form" onSubmit={handleSubmit(onSubmit)} noValidate>
+          <label className="form__label" htmlFor="email">
+            {t('login.labels.email')}
+          </label>
+          <Input
+            register={register}
```
Comment (alejobh) [inline src/pages/Login/index.jsx:71]: i think there should be validations for this and the password field, isn't it?

File: .nvmrc (hunk 2 above)
Comment (FlaviaCohen) [reply .nvmrc]: Are you talking about the space? I think it's a prettier thing.

File: src/index.jsx (hunk 3 above)
Comment (FlaviaCohen) [reply src/index.jsx]: You are right. I've already deleted that extension and left just `./App`

File: src/pages/Login/index.jsx (hunk 4 above)
Comment (FlaviaCohen) [reply src/pages/Login/index.jsx:71]: There are!

File: .nvmrc (hunk 2 above)
Comment (FlaviaCohen) [reply .nvmrc]: Reverted!

**Pull request merged**